components/src/androidTest/java/uk/gov/android/ui/components/m3/ButtonsM3Test.kt

```diff
@@ -11,19 +11,15 @@ import org.junit.Assert
 import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
-import org.junit.runner.RunWith
-import org.junit.runners.Parameterized
 import uk.gov.android.ui.components.m3.buttons.ButtonParameters
 import uk.gov.android.ui.components.m3.buttons.ButtonProvider
 import uk.gov.android.ui.components.m3.buttons.GdsButton
 import uk.gov.android.ui.theme.m3.GdsTheme
 import uk.gov.android.ui.theme.minimumTouchTarget
 
-@RunWith(Parameterized::class)
-class ButtonsM3Test(
-    private val parameters: ButtonParameters
-) {
+class ButtonsM3Test {
     private val expectedParameterSize = 6
+    private val parametersList = ButtonProvider().values.map { it.first() }.toList()
 
```

Comment:
Just out of curiosity, why are we using .map here and above, it's converting the values sequence directly into a list:
`ButtonProvider().values.toList()`

Reply:
Yeah, this does look quite counter intuative and I agree. The reason it's required is because the `ButtonProvider().value` is a `Sequence<List<params>>`, so running `ButtonProvider().value.toList()` produces a `List<List<params>>` 😭 
Probably we need to change the way the providers are setup